src/transformers/generation_utils.py

```diff
@@ -2549,6 +2563,7 @@ def top_k_top_p_filtering(
     logits: torch.FloatTensor,
     top_k: int = 0,
     top_p: float = 1.0,
+    tfs: float = 1.0,
```

Comment:
Could we maybe not include `tfs` in the `top_k_top_p_filtering` as the method should only include top_k and top_p. Would it be possible to apply it outside of this function instead?